SteamKit2/SteamKit2/Steam/Handlers/SteamAuthentication/SteamAuthentication.cs

```diff
@@ -169,7 +171,7 @@ public async Task<AuthPollResult> StartPolling( CancellationToken? cancellationT
                 // simply poll until confirmation is accepted, or whether they want to fallback to the next preferred confirmation type.
                 if ( Authenticator != null && preferredConfirmation.confirmation_type == EAuthSessionGuardType.k_EAuthSessionGuardType_DeviceConfirmation )
                 {
-                    var prefersToPollForConfirmation = await Authenticator.AcceptDeviceConfirmation();
+                    var prefersToPollForConfirmation = await Authenticator.AcceptDeviceConfirmationAsync();
```

Comment:
All of these `await` calls should have `.ConfigureAwait(false)` so that .NET Framework consumers don't end up deadlocking on a UI thread or ASP.NET execution context.

Not sure if/how it works on WinForms/WPF Core or MAUI.